geoh5py/objects/object_base.py

```diff
@@ -609,6 +612,24 @@ def add_default_visual_parameters(self):
 
         return self._visual_parameters
 
+    def remove_data_from_groups(self, data: list[Data] | Data) -> None:
+        """
+        Remove data children to all
+        :obj:`~geoh5py.groups.property_group.PropertyGroup` of the object.
+
+        :param data: :obj:`~geoh5py.data.data.Data` object,
+            :obj:`~geoh5py.shared.entity.Entity.uid` or
+            :obj:`~geoh5py.shared.entity.Entity.name` of data.
+        """
+        if not isinstance(data, list):
+            data = [data]
+
+        if not self.property_groups:
```

Comment:
self._property_groups ?

Reply:
Probably yes, mostly for consistency with Concatenated class.
Done